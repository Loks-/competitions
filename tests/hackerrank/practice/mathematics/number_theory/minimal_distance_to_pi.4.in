585507095600429 585507776729118
