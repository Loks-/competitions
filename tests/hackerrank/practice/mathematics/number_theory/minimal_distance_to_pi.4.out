1839425924481003/585507456665062
